kong/plugins/jwt/handler.lua

```diff
@@ -16,14 +16,14 @@ JwtHandler.PRIORITY = 1000
 -- Checks for the JWT in URI parameters, then in the `Authorization` header.
 -- @param request ngx request object
 -- @param conf Plugin configuration
--- @return token JWT token contained in request or nil
+-- @return token JWT token contained in request (can be a table) or nil
 -- @return err
 local function retrieve_token(request, conf)
   local uri_parameters = request.get_uri_args()
 
   for _, v in ipairs(conf.uri_param_names) do
     if uri_parameters[v] then
-      return uri_parameters[v]
+      return uri_parameters[v]  -- TODO: check multiple values, then this returns a table instead of a string
```

Comment:
@thibaultCha this happens when there are mutiple tokens in the uri. What to do? error out, or accept the first and discard the others?